src/screens/BackupSheet.js

```diff
@@ -177,7 +188,9 @@ const BackupSheet = () => {
             onSecondaryAction={onAlreadyBackedUp}
             primaryLabel="Back up now"
             secondaryLabel="􀁣 Already backed up"
-            titleText="Back up your wallets"
+            titleText={`Back up your wallet${backupeableWalletsCount > 1 &&
+              's'}`}
```

Comment:
this can return `Back up your walletfalse` if backupeableWalletsCount is 0

it should instead be : 
```
titleText={`Back your wallet${backupeableWalletsCount > 1 ? 's' : ''}`}
```